src/Mutator/Util/Exception/MutatorException.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator\Util\Exception;
+
+use Infection\Mutator\Util\Mutator;
+
+/**
+ * @internal
+ */
+final class MutatorException extends \Exception
```

Comment:
why is this class in the `Util` namespace?

Reply:
It felt like the best place for it, but i'm open to suggestions.

Reply:
The `Exception` namespace but without `Util`

Reply:
I don't want to use the `Exception` namespace within `Mutator` for actual Exceptions. The Throw and Finally_ mutator would be better suited to live in that namespace.

Reply:
Yes indeed, sorry I was thinking of `src/Exception`